wsi_superpixel_guided_labeling/web_client/views/body/ActiveLearningView.js

```diff
@@ -657,13 +668,15 @@ const ActiveLearningView = View.extend({
         const data = this.generateClassificationJobData();
         data.jobId = this.lastRunJobId;
         data.randomInput = false;
+        const excluded = _.map(store.exclusions, (idx) => store.categories[idx + 1].label);
         restRequest({
             method: 'POST',
             url: `slicer_cli_web/${this.activeLearningJobUrl}/rerun`,
             data: {
                 jobId: this.lastRunJobId,
                 randominput: false,
-                train: true
+                train: true,
+                exclude: excluded
```

Comment:
I think this needs to be `exclude: JSON.stringify(excluded)`.

Reply:
Oof, yes. Fixed now, thank you!